catalyst-generator/src/main/resources/templates/camel/router/PGPRouter.java

```diff
@@ -0,0 +1,53 @@
+{{=<% %>=}}
+
+<%license%>
+
+package <%packageName%>;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+/**
+ * Sample Camel route to demonstrate encryption and decryption using PGP
+ */
+@Component
+public class PGPRouter extends RouteBuilder {
+
+  @Value("${catalyst.pgp.privateKeyFile}")
+  private String privateKeyFile;
```

Comment:
Better to use Property class like in sftp sample with constructor property injection.

Reply:
IMO, for simple string-based Camel properties it is okay to embed them into route definition, ti keep everything simple